test/e2e/ganache.js

```diff
@@ -23,6 +23,27 @@ class Ganache {
     return this._server.provider;
   }
 
+  async getAccounts() {
+    return await this.getProvider().request({
+      method: 'eth_accounts',
+      params: [],
+    });
+  }
+
+  async getBalance() {
+    const accounts = await this.getAccounts();
+    const balanceHex = await this.getProvider().request({
+      method: 'eth_getBalance',
+      params: [accounts[0], 'latest'],
+    });
+    const balanceInt = parseInt(balanceHex, 16) / 10 ** 18;
+
+    const balanceFormatted =
+      balanceInt % 1 === 0 ? balanceInt : balanceInt.toFixed(4);
```

Comment:
If the balance is a float number, we round it up to 4 decimals to match UI.
If it's an integer, do nothing